### DISCUSSION TITLE
will infinity use all available cpus? or would I rather just give it one cpu and run multiple instances?

### DISCUSSION THREAD

**Discussion by nikcaryo-super** [michaelfeil/infinity#191] [Q&A] [answered]
just trying to deploy an embedding server for sentence transformers models without reinventing the wheel.

**Comment by michaelfeil** [comment, marked as answer]
Yes - why not try it out?

`optimum` (`onnx`) / ct2 / torch are using all cores.  Don't be shocked, the default setting in these libs will be all physical cores (aka w/o hyperthreading), which should be a good choice.
